proto/iavl/iavl_api.proto

```diff
@@ -283,6 +296,12 @@ message HasResponse {
 message GetResponse {
   int64 index = 1;
   bytes value = 2;
+  bool missing = 3;
```

Comment:
how about changing to more standard `not_found` (like in HTTP) or `exists` (like in SQL)?

Reply:
Sure, `not_found` sounds good. I think it makes sense for the zero value to default to the typical case where the key exists.